conn.go

```diff
@@ -846,6 +862,12 @@ func (c *Conn) WriteCompressedMessages(codec CompressionCodec, msgs ...Message)
 					if err == nil && p.ErrorCode != 0 {
 						err = Error(p.ErrorCode)
 					}
+					if err == nil {
+						partition = p.Partition
+						offset = p.Offset
+						appendTime = time.Unix(p.Timestamp, 0)
```

Comment:
The Kafka protocol documentation seems to indicate that the timestamp is expressed as a number of milliseconds since the unix epoch, but the code assumes it is seconds, or am I missing something?

https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-ProduceAPI

Reply:
Interesting, you are probably right.
I read this documentation here: https://kafka.apache.org/protocol#The_Messages_Produce
Not sayd about milliseconds )